5calls/app/src/main/res/values/strings.xml

```diff
@@ -70,6 +70,18 @@
     <!-- Button that allows users to send 5 Calls an email [CHAR_LIMIT=30] -->
     <string name="contact_us_btn">contact us</string>
 
+    <!-- Prompt to connect with 5 Calls through social media [CHAR_LIMIT=100] -->
+    <string name="social_prompt">You can also get in touch via</string>
+
+    <!-- Button that links to 5 Calls' Twitter -->
+    <string name="twitter_btn">Twitter</string>
```

Comment:
Should these 3 buttons be translatable? Are the words "Facebook"/"Twitter"/"Instagram" the same in all languages? If not, put translatable=false in the string tag.